File: src/test/resources/book-source-code/java8ForImpatient/ch4/sec03/SliderDemo.java
import javafx.scene.Scene;
import javafx.scene.control.Label;
import javafx.scene.control.Slider;
import javafx.scene.layout.VBox;
import javafx.scene.text.Font;
import javafx.stage.Stage;

public class SliderDemo extends Application {
    public void start(Stage stage) {
        Label message = new Label("Hello, JavaFX!");
        message.setFont(new Font(100));
        Slider slider = new Slider();
        slider.setValue(100);
        slider.valueProperty().addListener(property
            -> message.setFont(new Font(slider.getValue())));
        VBox root = new VBox();
        root.getChildren().addAll(slider, message);
        Scene scene = new Scene(root);
        stage.setTitle("Hello");
        stage.setScene(scene);
        stage.show();
    }
}

File: src/test/resources/book-source-code/java8ForImpatient/ch7/sec13/sliderdemo.js
var message = new javafx.scene.control.Label("Hello, JavaFX!");
var Font = javafx.scene.text.Font
message.font = new Font(100);
var slider = new javafx.scene.control.Slider()
slider.value = 100
slider.valueProperty().addListener(
   new javafx.beans.InvalidationListener(function(property) 
      message.font = new Font(slider.value)))
var root = new javafx.scene.layout.VBox()
root.children.addAll(slider, message)
$STAGE.scene = new javafx.scene.Scene(root);
$STAGE.title = "Hello";

/*
*/

